### DISCUSSION TITLE
`automatic_installation` doesn't seem to be working

### DISCUSSION THREAD

**Discussion by nathanielevan** [mason-org/mason.nvim#997] [Q&A] [answered]
Relevant parts of my config:
- [packer.nvim config](https://github.com/nathanielevan/dotfiles/blob/master/.config/nvim/lua/plugins/init.lua#L84)
- [lsp config](https://github.com/nathanielevan/dotfiles/tree/master/.config/nvim/lua/plugins/config/lsp)

As described in the title, `automatic_installation` doesn't seem to work with my config; however `ensure_installed` works fine, I can install my LSP servers manually and they seem to function properly. I'm not sure what I'm doing wrong; some pointers would be much appreciated!

**Comment by williamboman** [comment]
Hey! What do you mean by doesn't seem to be working?

**Reply by nathanielevan** [reply]
Never mind actually, I figured it out, my bad. `automatic_installation` didn't seem to be installing the LSP servers it's supposed to install automatically (i.e. the servers I configured in `lspconfig`), that was simply it. It appears removing the `module` keyword from `lspconfig` in my `packer.nvim` config fixed the issue, so I think I can close this for now.

**Comment by nathanielevan** [comment, marked as answer]
Fixed: removing [this line](https://github.com/nathanielevan/dotfiles/blob/235b474477a7ec4135dc1ae52479fd9207c0e22e/.config/nvim/lua/plugins/init.lua#L93) fixed my issue above.
